src/Text/Pretty/Simple.hs

```diff
@@ -426,9 +426,10 @@ pStringNoColor = pStringOpt defaultOutputOptionsNoColor
 -- ( 1
 -- ,
 --     ( 2
---     , "foo
---       bar
---       baz"
+--     ,
+--         "foo
+--         bar
+--         baz"
```

Comment:
For this, and the next test, this behaviour was simply easier to implement. I'm sure we could find some way to get the old behaviour back if there's a strong preference.

Reply:
Personally, I did like the compact old output.

Reply:
@georgefst I agree with sjakobi.  I like the more compact old layout here.  Could you change this so the old output style is preserved?

Reply:
Done in c1b084d. Actually turned out to be very simple.